src/theme/Buttons.js

```diff
@@ -1,44 +0,0 @@
-import chroma from 'chroma-js';
```

Comment:
I think we can keep the thene, just need to figure out how to roll it into tailwind

Reply:
wouldn't a different tailwinds.js file be equivalent?